src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -130,6 +131,10 @@ define(function (require, exports, module) {
             
             this._forceVisible();
             this.getFlux().actions.export.addAssetThrottled(document, documentExports, selectedLayers, props);
+            
+            if (preset) {
+                headlights.logEvent("export", "preset", preset);
```

Comment:
I'm not sure if preset is a string here. Presumably it is, but there isn't any JSDoc. Can you confirm that it's a string and, if so, add the `param` annotation for this method?